### DISCUSSION TITLE
Orchard core with static site generators?

### DISCUSSION THREAD

**Discussion by 1alikhan** [OrchardCMS/OrchardCore#12234] [Q&A]
Hello, can some one please clarify what would be the best way to use a static site generator with orchard core? I remember, i saw a talk of Sebastian on this topic a while back but cant find that anymore. It was about integration of orchard core with statiq or is there a better way?

Help would be appreciated. Thanks.

**Comment by hishamco** [comment]
Please check Lombiq YouTube channel I'm sure it's there

**Reply by hishamco** [reply]
Check if you can find it [here](https://www.youtube.com/watch?v=yWpz8p-oaKg&list=PLuskKJW0FhJf_uqOFbozmquvoKeSgKQG2&index=3)

**Reply by agriffard** [reply]
OC as a Decoupled CMS (which is the approach shown in the video) is not the same thing as OC as a Headless CMS that is connected to a static site generator to retrieve its content.
You must be talking about this demo: https://youtu.be/eaTX0alFF2M?t=2978
